site/content/docs/4.3/components/card.md

```diff
@@ -601,88 +604,94 @@ Just like with card groups, card footers in decks will automatically line up.
 </div>
 {{< /example >}}
 
-### Card columns
+### Card columns (Masonry layout)
 
-Cards can be organized into [Masonry](https://masonry.desandro.com/)-like columns with just CSS by wrapping them in `.card-columns`. Cards are built with CSS `column` properties instead of flexbox for easier alignment. Cards are ordered from top to bottom and left to right.
+In `v4` we used a CSS-only technique to mimic the behaviour of [Masonry](https://masonry.desandro.com/)-like columns, but this technique came with lots of unpleasant [side effects](https://github.com/twbs/bootstrap/pull/28922) so in `v5` we decided to just use the wonderful Masonry plugin. **Masonry is not included in Bootstrap**, but can easily be added by adding the following javascript file:
```

Comment:
I would end the sentence after "side effects", it becomes too long:

...came with lots of unpleasant side effects. So in `v5`...

Reply:
Thanks, I've edited the copy.